4-perftools/Dockerfile.rhel7

```diff
@@ -38,7 +37,7 @@ USER 0
 
 WORKDIR ${HOME}
 
-COPY contrib/etc/scl_enable /root/etc/scl_enable
+COPY root/root/etc/scl_enable /root/etc/scl_enable
```

Comment:
`USER 0` is default, so this command is redundant too and might not affect the COPY.

>  I think that it should be removed as part of a different PR so as not to muddy the history.

OK. I thought it could be, because it related to getting files into container. And aim of `root/` folder is to be able to add all files in once (in almost all cases).